packages/iTwinUI-react/src/core/Slider/Slider.tsx

```diff
@@ -174,6 +183,13 @@ export type SliderProps = {
    * high-volume of updates will occur when dragging.
    */
   onUpdate?: (values: ReadonlyArray<number>) => void;
+  /**
+   * The orientation of slider
+   * 'horizontal' - Slider is horizontal and fills from left to right. Default.
+   * 'vertical'- Slider is vertical and fills from bottom to top.
+   * @default 'horizontal'
+   */
```

Comment:
I'd remove the two comments describing what horizontal and vertical are

Reply:
**[DONE]** Yes, that makes sense, removed lines 188 and 189